### ISSUE TITLE
Add hpo xref to rheumatoid arthritis

### ISSUE THREAD

**Issue by klaasMensaert** [monarch-initiative/mondo#6094]
**Mondo term (ID and Label):**
MONDO:0008383  - rheumatoid arthritis

**Xref that should be fixed (ID and label):**
HP:0001370 - Rheumatoid arthritis

**Your nano-attribution (ORCID)**
0000-0001-9859-8589

**Other comments:**
xref url: https://hpo.jax.org/app/browse/term/HP:0001370
